pom.xml

```diff
@@ -334,7 +334,7 @@
         <version.org.jboss.metadata>13.0.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>1.4.1.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>5.9.5.Final</version.org.jboss.narayana>
-        <version.org.jboss.openjdk-orb>8.1.3.Final</version.org.jboss.openjdk-orb>
```

Comment:
What is the purpose of this component upgrade?

Reply:
Not my change. Which one? openjdk-orb?

Reply:
I don't know how that got in there ...

Reply:
This is going to mess with git history if merged.  Can you fix your branch?